INFO - 2025-02-06 03:50:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-06 03:53:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-06 03:53:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-06 03:53:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-02-06 03:57:38 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\sisOdon\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\sisOdon\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'db_odontologia', 3306, '', 0)
#1 C:\xampp\htdocs\sisOdon\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\sisOdon\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\sisOdon\app\Models\ModeloGeneral.php(92): CodeIgniter\Database\BaseConnection->query('CALL SP_INSERT_...', Array)
#4 C:\xampp\htdocs\sisOdon\app\Controllers\CCasos.php(28): App\Models\ModeloGeneral->MetodoModeloInsertCaso(Array)
#5 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(933): App\Controllers\CCasos->MetodoInsertarCaso()
#6 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CCasos))
#7 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\sisOdon\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\sisOdon\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\sisOdon\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\sisOdon\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\sisOdon\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\sisOdon\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\sisOdon\app\Models\ModeloGeneral.php(92): CodeIgniter\Database\BaseConnection->query('CALL SP_INSERT_...', Array)
#3 C:\xampp\htdocs\sisOdon\app\Controllers\CCasos.php(28): App\Models\ModeloGeneral->MetodoModeloInsertCaso(Array)
#4 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(933): App\Controllers\CCasos->MetodoInsertarCaso()
#5 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CCasos))
#6 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\sisOdon\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\sisOdon\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\sisOdon\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 {main}
INFO - 2025-02-06 03:57:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-02-06 03:57:42 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\sisOdon\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\sisOdon\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'db_odontologia', 3306, '', 0)
#1 C:\xampp\htdocs\sisOdon\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\sisOdon\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\sisOdon\app\Models\ModeloGeneral.php(103): CodeIgniter\Database\BaseConnection->query('CALL SP_SELECT_...')
#4 C:\xampp\htdocs\sisOdon\app\Controllers\CCasos.php(44): App\Models\ModeloGeneral->SelectCasosFM()
#5 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(933): App\Controllers\CCasos->SelectCasosFC()
#6 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CCasos))
#7 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\sisOdon\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\sisOdon\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\sisOdon\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\sisOdon\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\sisOdon\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\sisOdon\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\sisOdon\app\Models\ModeloGeneral.php(103): CodeIgniter\Database\BaseConnection->query('CALL SP_SELECT_...')
#3 C:\xampp\htdocs\sisOdon\app\Controllers\CCasos.php(44): App\Models\ModeloGeneral->SelectCasosFM()
#4 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(933): App\Controllers\CCasos->SelectCasosFC()
#5 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CCasos))
#6 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\sisOdon\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\sisOdon\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\sisOdon\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 {main}
CRITICAL - 2025-02-06 03:57:42 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
[Method: GET, Route: SelectCasos]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Models\ModeloGeneral.php(103): CodeIgniter\Database\BaseConnection->query('CALL SP_SELECT_CASO')
 3 APPPATH\Controllers\CCasos.php(44): App\Models\ModeloGeneral->SelectCasosFM()
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\CCasos->SelectCasosFC()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CCasos))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-02-06 03:59:13 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\sisOdon\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\sisOdon\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'db_odontologia', 3306, '', 0)
#1 C:\xampp\htdocs\sisOdon\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\sisOdon\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\sisOdon\app\Models\ModeloGeneral.php(103): CodeIgniter\Database\BaseConnection->query('CALL SP_SELECT_...')
#4 C:\xampp\htdocs\sisOdon\app\Controllers\CCasos.php(44): App\Models\ModeloGeneral->SelectCasosFM()
#5 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(933): App\Controllers\CCasos->SelectCasosFC()
#6 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CCasos))
#7 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\sisOdon\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\sisOdon\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\sisOdon\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\sisOdon\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\sisOdon\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\sisOdon\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\sisOdon\app\Models\ModeloGeneral.php(103): CodeIgniter\Database\BaseConnection->query('CALL SP_SELECT_...')
#3 C:\xampp\htdocs\sisOdon\app\Controllers\CCasos.php(44): App\Models\ModeloGeneral->SelectCasosFM()
#4 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(933): App\Controllers\CCasos->SelectCasosFC()
#5 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CCasos))
#6 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\sisOdon\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\sisOdon\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\sisOdon\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 {main}
CRITICAL - 2025-02-06 03:59:13 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
[Method: GET, Route: SelectCasos]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Models\ModeloGeneral.php(103): CodeIgniter\Database\BaseConnection->query('CALL SP_SELECT_CASO')
 3 APPPATH\Controllers\CCasos.php(44): App\Models\ModeloGeneral->SelectCasosFM()
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\CCasos->SelectCasosFC()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CCasos))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-02-06 04:00:54 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\sisOdon\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\sisOdon\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'db_odontologia', 3306, '', 0)
#1 C:\xampp\htdocs\sisOdon\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\sisOdon\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\sisOdon\app\Models\ModeloGeneral.php(103): CodeIgniter\Database\BaseConnection->query('CALL SP_SELECT_...')
#4 C:\xampp\htdocs\sisOdon\app\Controllers\CCasos.php(44): App\Models\ModeloGeneral->SelectCasosFM()
#5 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(933): App\Controllers\CCasos->SelectCasosFC()
#6 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CCasos))
#7 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\sisOdon\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\sisOdon\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\sisOdon\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\sisOdon\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\sisOdon\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\sisOdon\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\sisOdon\app\Models\ModeloGeneral.php(103): CodeIgniter\Database\BaseConnection->query('CALL SP_SELECT_...')
#3 C:\xampp\htdocs\sisOdon\app\Controllers\CCasos.php(44): App\Models\ModeloGeneral->SelectCasosFM()
#4 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(933): App\Controllers\CCasos->SelectCasosFC()
#5 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CCasos))
#6 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\sisOdon\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\sisOdon\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\sisOdon\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 {main}
CRITICAL - 2025-02-06 04:00:54 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
[Method: GET, Route: SelectCasos]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Models\ModeloGeneral.php(103): CodeIgniter\Database\BaseConnection->query('CALL SP_SELECT_CASO')
 3 APPPATH\Controllers\CCasos.php(44): App\Models\ModeloGeneral->SelectCasosFM()
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\CCasos->SelectCasosFC()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CCasos))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-02-06 04:02:08 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\sisOdon\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\sisOdon\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'db_odontologia', 3306, '', 0)
#1 C:\xampp\htdocs\sisOdon\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\sisOdon\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\sisOdon\app\Models\ModeloGeneral.php(92): CodeIgniter\Database\BaseConnection->query('CALL SP_INSERT_...', Array)
#4 C:\xampp\htdocs\sisOdon\app\Controllers\CCasos.php(28): App\Models\ModeloGeneral->MetodoModeloInsertCaso(Array)
#5 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(933): App\Controllers\CCasos->MetodoInsertarCaso()
#6 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CCasos))
#7 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\sisOdon\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\sisOdon\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\sisOdon\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\sisOdon\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\sisOdon\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\sisOdon\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\sisOdon\app\Models\ModeloGeneral.php(92): CodeIgniter\Database\BaseConnection->query('CALL SP_INSERT_...', Array)
#3 C:\xampp\htdocs\sisOdon\app\Controllers\CCasos.php(28): App\Models\ModeloGeneral->MetodoModeloInsertCaso(Array)
#4 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(933): App\Controllers\CCasos->MetodoInsertarCaso()
#5 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CCasos))
#6 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\sisOdon\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\sisOdon\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\sisOdon\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 {main}
INFO - 2025-02-06 04:02:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-02-06 04:02:18 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\sisOdon\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\sisOdon\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'db_odontologia', 3306, '', 0)
#1 C:\xampp\htdocs\sisOdon\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\sisOdon\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\sisOdon\app\Models\ModeloGeneral.php(103): CodeIgniter\Database\BaseConnection->query('CALL SP_SELECT_...')
#4 C:\xampp\htdocs\sisOdon\app\Controllers\CCasos.php(44): App\Models\ModeloGeneral->SelectCasosFM()
#5 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(933): App\Controllers\CCasos->SelectCasosFC()
#6 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CCasos))
#7 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\sisOdon\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\sisOdon\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\sisOdon\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\sisOdon\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\sisOdon\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\sisOdon\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\sisOdon\app\Models\ModeloGeneral.php(103): CodeIgniter\Database\BaseConnection->query('CALL SP_SELECT_...')
#3 C:\xampp\htdocs\sisOdon\app\Controllers\CCasos.php(44): App\Models\ModeloGeneral->SelectCasosFM()
#4 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(933): App\Controllers\CCasos->SelectCasosFC()
#5 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CCasos))
#6 C:\xampp\htdocs\sisOdon\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\sisOdon\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\sisOdon\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\sisOdon\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 {main}
CRITICAL - 2025-02-06 04:02:18 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
[Method: GET, Route: SelectCasos]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Models\ModeloGeneral.php(103): CodeIgniter\Database\BaseConnection->query('CALL SP_SELECT_CASO')
 3 APPPATH\Controllers\CCasos.php(44): App\Models\ModeloGeneral->SelectCasosFM()
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\CCasos->SelectCasosFC()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CCasos))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-06 15:50:28 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "VistaSelect.php"
[Method: GET, Route: Select]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('VistaSelect.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('VistaSelect', [], true)
 3 APPPATH\Controllers\Home.php(63): view('VistaSelect', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->SelectUsuarioFC()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2025-02-06 16:03:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-02-06 16:57:30 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "login.php"
[Method: GET, Route: login]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('login', [], true)
 3 APPPATH\Controllers\Home.php(75): view('login')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-06 16:57:40 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "login.php"
[Method: GET, Route: /]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('login', [], true)
 3 APPPATH\Controllers\Home.php(75): view('login')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-06 16:58:42 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "login.php"
[Method: GET, Route: /]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('login', [], true)
 3 APPPATH\Controllers\Home.php(75): view('login')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->vistalogin()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-06 17:10:03 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "login.php"
[Method: GET, Route: /]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('login', [], true)
 3 APPPATH\Controllers\Home.php(75): view('login')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->vistalogin()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-06 17:11:56 --> CodeIgniter\Database\Exceptions\DatabaseException: You must set the database table to be used with your query.
[Method: POST, Route: login]
in SYSTEMPATH\Database\BaseConnection.php on line 927.
 1 SYSTEMPATH\Model.php(716): CodeIgniter\Database\BaseConnection->table(null)
 2 SYSTEMPATH\Model.php(922): CodeIgniter\Model->builder()
 3 APPPATH\Controllers\Home.php(84): CodeIgniter\Model->__call('obtenerUsuarioPorCorreo', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->verificacionlogin()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2025-02-06 17:27:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-06 17:27:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-06 21:00:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-06 21:01:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-06 21:01:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-06 21:02:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-06 21:02:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-06 21:08:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-06 21:09:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-06 21:15:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-06 21:24:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-06 21:27:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-06 21:31:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-06 21:32:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-02-06 21:49:10 --> TypeError: count(): Argument #1 ($value) must be of type Countable|array, null given
[Method: POST, Route: verificacion]
in APPPATH\Controllers\Home.php on line 84.
 1 APPPATH\Controllers\Home.php(84): count(null)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->pruebalogin()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2025-02-06 21:51:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-06 22:01:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
